roles/openshift_cli/library/container_binary_sync.py

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# vim: expandtab:tabstop=4:shiftwidth=4
+# pylint: disable=missing-docstring
+#
+
+import random
+import tempfile
+import shutil
+import subprocess
+import os.path
+
+from ansible.module_utils.basic import AnsibleModule
+
+
+DOCUMENTATION = '''
+---
+module: container_binary_sync
+short_description: Copies OpenShift binaries out of the given image tag to host system.
+'''
+
+
+class BinarySyncError(Exception):
+    def __init__(self, msg, cmd, exit_code, output):
+        super(BinarySyncError, self).__init__(msg)
+        self.msg = msg
+        self.cmd = cmd
+        self.exit_code = exit_code
+        self.output = output
+
+
+def run_cmd(args):
```

Comment:
There is actually a helper method in ansible module_utils for this already: https://github.com/ansible/ansible/blob/devel/lib/ansible/module_utils/basic.py#L1951